README.md

```diff
@@ -10,6 +10,8 @@ Zeplin for Visual Studio Code lets you quickly access the designs you’re worki
 
 ## Usage
 
+![Preview](resources/docs/sample.gif)
```

Comment:
It might be better to add this before the "Installation" header, as an introduction after the initial description.